### ISSUE TITLE
Add optimal stretch value to getRecommendedFit

### ISSUE THREAD

**Issue by nomasi** [SizeMeCom/sizeme-react#143]
In cases where the optimalFit value is 1000 and the item stretches a lot, many sizes will have a totalFit value of 1000 and thus be a possible candidate for the recommended size.  This leads to the recommended size being random.  In these cases, the max componentStretch value should be the deciding factor.  

Proposed change for the getRecommendedFit function:
- count the maxComponentStretch value first
  - is the max single value of all the componentStretch values in the item's matchMap
- add a DEFAULT_OPTIMAL_STRETCH value, can be 33 at first; maybe same handling as optimalFit --> optStretch
- change the newFit calc to something like:
```
const newFit = (Math.abs(res.totalFit - optFit) * 100) + Math.abs(maxComponentStretch - optStretch);
```

The idea is to have the stretch calc come into play only if two sizes have an identical distance from the optimal fit value.  If this is done by changing the newFit value by x 100, I don't know if that messes up other handling as the fit value is also returned from the function.

**Comment by nomasi** [comment]
Note!  This feature is dependent on client feedback, so it shouldn't be done yet.  I just wanted to write the proposal down if the client green lights it.  :)

**Comment by nomasi** [comment]
The DEFAULT_OPTIMAL_STRETCH can be a local static value in this phase as it's a fringe use case with 0-1 clients using it.

**Comment by nomasi** [comment]
The DEFAULT_OPTIMAL_STRETCH can be **5** at this point and this should be done now as we have a client (albeit a small one) that needs it.

**Comment by nomasi** [comment]
Also; the Slider functionality should be updated if the optimalFit value is 1000.  Currently, there's not much information there.  I'll prepare specs for that change shortly.

**Comment by nomasi** [comment]
Test product for this:

https://sizeme.com/api/products/PERFURO-TEST-JERSEY-MEN

**Comment by nomasi** [comment]
Fixed via #146
